backend/Rhyolite/Backend/Logging.hs

```diff
@@ -105,6 +105,8 @@ data LoggingContext m = LoggingContext
 data RhyoliteLogAppender
    = RhyoliteLogAppender_Stderr   RhyoliteLogAppenderStderr
    | RhyoliteLogAppender_File     RhyoliteLogAppenderFile
+   | RhyoliteLogAppender_Filerotate RhyoliteLogAppenderFileRotate
+   -- fastlogger also supports time based rotation, but it's API calls for a function which must parse a pair of "human readable" ByteStrings as dates.  A buggy implementation of half of common lisp in json syntax is out of scope for this yakshave.
```

Comment:
s/it's API/its API/

Reply:
Can we also use camel case? `FileRotate`? You may say, but `Stderr`! `stderr` (all one case) appears everywhere and is not weird. But `filerotate` is much less obvious to the eyes.

Reply:
It also makes it match `RhyoliteLogAppenderFileRotate` better.